articles/client-platforms/react/08-calling-apis.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Calling APIs
+description: This tutorial will show you how to make authenticated api calls with ReactJS.
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/05-Calling-Api',
```

Comment:
@flaviogranero 
Update sample URL please, the valid one is: https://github.com/auth0-samples/auth0-react-sample/tree/master/08-Calling-Api